src/linear_algebra/affine_space.lean

```diff
@@ -15,19 +15,71 @@ open_locale classical
 # Affine spaces
 
 This file defines affine spaces (over modules) and subspaces, affine
-maps, affine combinations of points, and the affine span of a set of
-points.
+maps, affine combinations of points, affinely independent families of
+points, and the affine span of a set of points.
+
+## Main definitions
+
+* `affine_space k V P` is an abbreviation for `add_torsor V P` in the
+  case of `module k V`.  Definitions and results not depending on the
```

Comment:
Could you mention here what the variables `k`, `V` and `P` represent? Something like: "`P` is the type of points in the space and `V` the `k`-module of displacement vectors".

Reply:
Added that suggested text.